.github/workflows/check-licenses.sh

```diff
@@ -44,6 +44,7 @@ while read line; do
       [[ $line == *"(The MIT License)"* ]] ||
       [[ $line == *"(MIT License)"* ]] ||
       [[ $line == *"(MIT)"* ]] ||
+      [[ $line == *"(Unicode/ICU License)"* ]] ||
```

Comment:
Did we get confirmation that it's ok to use that license?

Reply:
Hi Martin,

Not yet, Hauke asked me to prepare PR and he will resolve it once when he gets the confirmation for the license.
Hauke proposed to make the PR on develop branch.